fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessor.java

```diff
@@ -392,7 +392,7 @@ private void refreshExternalTable(TaskRunContext context) {
         for (Pair<BaseTableInfo, Table> tablePair : snapshotBaseTables.values()) {
             BaseTableInfo baseTableInfo = tablePair.first;
             Table table = tablePair.second;
-            if (!table.isNativeTableOrMaterializedView()) {
+            if (!table.isNativeTableOrMaterializedView() && !table.isHiveView()) {
```

Comment:
What's behavior of refresh hive view? exception?

Reply:
it is not supported now in hive catalog, will be supported in the future.

Reply:
now it throws exception.